src/UI/AppLayout.jsx

```diff
@@ -16,11 +11,13 @@ function AppLayout() {
   };
 
   return (
-    <StyledAppLayout>
-      <Header toggleModal={toggleModal} />
+    <div className="box-border w-full">
+      <Header toggleModal={toggleModal} className="sticky top-0" />
+
       <NewsSourceModal isOpen={isModalOpen} toggleModal={toggleModal} />
-      <Outlet />
-    </StyledAppLayout>
+
+      <Outlet className="min-h-screen min-w-full" />
```

Comment:
Same idea here for classes. Actually, not even sure if the Outlet has a root component which this className can be attached, it's possible that it is just a bunch our Route components all on the same level.

So what you can do here if you need these styles, you can wrap the Outlet inside of a div, something like this:

```js
<div className="min-h-screen min-w-full">
  <Outlet />
</div>
```